### PR TITLE
refac: Rename `grid_area` to `grid_area_code`

### PR DESCRIPTION
**Pull request by BjarkeMeier**
# Description

## Pull-request quality

- [x] The title adheres to [this guide](https://github.com/Mech0z/GitHubGuidelines)
- [ ] Tests are written and executed locally
- [ ] Subsystem tests have been tested (by manually deploying to `dev_002`)
- [ ] Documentation has been updated
- [ ] The integration [event catalog](https://energinet.atlassian.net/wiki/spaces/D3/pages/555581556/Event+catalog) has been updated
- [ ] C4 diagrams have been updated and Team Outlaws has been informed about relevant changes

### CODE REVIEW

File: source/databricks/calculation_engine/package/constants/energy_result_column_names.py [hunk 1]
```diff
@@ -18,7 +18,7 @@
 
 class EnergyResultColumnNames(ResultColumnNames):
     time_series_type = "time_series_type"
-    from_grid_area = "out_grid_area"
+    from_grid_area = "out_grid_area_code"
     """The delta table column name ought to be renamed to 'from_grid_area'.
```
Comment (energinet-ajw) [inline source/databricks/calculation_engine/package/constants/energy_result_column_names.py:22]: I guess this is a place where business language meets Wholesale language:
from_grid_area = "out_grid_area_code"

And is the comment still valid?

File: source/databricks/calculation_engine/tests/calculation/output/test_total_monthly_amounts_storage_model_factory.py [hunk 2]
```diff
@@ -201,14 +201,13 @@ def test__get_column_group_for_calculation_result_id__excludes_expected_other_co
         TotalMonthlyAmountsColumnNames.calculation_type,
         TotalMonthlyAmountsColumnNames.calculation_execution_time_start,
         TotalMonthlyAmountsColumnNames.calculation_result_id,
-        TotalMonthlyAmountsColumnNames.grid_area,
```
Comment (energinet-ajw) [inline source/databricks/calculation_engine/tests/calculation/output/test_total_monthly_amounts_storage_model_factory.py:204]: Is grid_area to be removed?

File: source/databricks/calculation_engine/tests/calculation/output/test_wholesale_storage_model_factory.py [hunk 3]
```diff
@@ -278,7 +278,6 @@ def test__get_column_group_for_calculation_result_id__excludes_expected_other_co
         WholesaleResultColumnNames.calculation_execution_time_start,
         WholesaleResultColumnNames.calculation_result_id,
         WholesaleResultColumnNames.amount_type,
-        WholesaleResultColumnNames.grid_area,
```
Comment (energinet-ajw) [inline source/databricks/calculation_engine/tests/calculation/output/test_wholesale_storage_model_factory.py:281]: Same question as previous.

**Review by energinet-ajw — APPROVED** [review]

File: source/databricks/calculation_engine/package/constants/energy_result_column_names.py (hunk 1 above)
Comment (BjarkeMeier) [reply source/databricks/calculation_engine/package/constants/energy_result_column_names.py:22]: No, I think it's a rename decision that wasn't carried out all the way as explained by the comment 😄

File: source/databricks/calculation_engine/tests/calculation/output/test_total_monthly_amounts_storage_model_factory.py (hunk 2 above)
Comment (BjarkeMeier) [reply source/databricks/calculation_engine/tests/calculation/output/test_total_monthly_amounts_storage_model_factory.py:204]: I think the grid_area was a mistake and the test only passed due to the error fixed below.
If you look in the sut, you'll also see that grid_area participates in forcing separate calculation result ids.

File: source/databricks/calculation_engine/tests/calculation/output/test_wholesale_storage_model_factory.py (hunk 3 above)
Comment (BjarkeMeier) [reply source/databricks/calculation_engine/tests/calculation/output/test_wholesale_storage_model_factory.py:281]: I think the grid_area was a mistake and the test only passed due to the error fixed below.
If you look in the sut, you'll also see that grid_area participates in forcing separate calculation result ids.

**Pull request merged**